topological_action_planner/src/topological_action_planner/util.py

```diff
@@ -0,0 +1,46 @@
+import networkx as nx
+
+from topological_action_planner_msgs.msg import Edge
+
+
+def generate_dummy_graph(n_nodes=10, n_edges=20) -> nx.Graph:
```

Comment:
So this is for testing only? Then it shouldn't be in the library

Reply:
Where then?

Reply:
In a test/example folder

Reply:
It's not an example, it is a utility function that is currently unused. It might be useful in a unit test though